sdk/cmake/host-tools.cmake

```diff
@@ -118,3 +118,17 @@ function(setup_host_tools)
         add_dependencies(native-${_module} host-tools ${INSTALL_DIR}/bin/${HOST_EXTRA_DIR}${_module}${HOST_MODULE_SUFFIX})
     endforeach()
 endfunction()
+
+function(generate_whitelist_code _whitelist_file)
```

Comment:
This belongs to `modules/rostests/CMakeLists.txt`

Reply:
IMO everything in here should go there.

Reply:
You have to be more specific on what "here" and "there" means for me to understand this last sentence :sweat_smile:

Reply:
Everything in this file should go to modules/rostests/CMakeLists.txt